### ISSUE TITLE
Staff Main possible whitepace issue.

### ISSUE THREAD

**Issue by EsenDR** [WickedWizard3588/YAGPDB-CC#4]
As the title suggests, there seems to be a possible whitespace issue with the main CC. Happens with PAG and YAG. Most, if not all commands return a "Custom command response was longer than 2k (contact an admin on the server...)" when used which is an issue because you dont know what it says.
![image](https://user-images.githubusercontent.com/52873478/106653998-674dbd80-654c-11eb-800e-ef4a4e443f2c.png)  
With PAG

![image](https://user-images.githubusercontent.com/52873478/106654137-949a6b80-654c-11eb-96de-906f707b406c.png)  
With YAG

If you want more info, feel free to dm me on discord: Esen Rurouni#2945

**Comment by WickedWizard3588** [comment]
Hey,
I tested it out myself. Both YAG and PAGST are working fine for me. Can you do some basic debugging for me, like adding plain text and giving some data regarding the config vars other than the ones in the picture?

Thanks!!

**Comment by EsenDR** [comment]
So I've tried to make some response lines in certain bits like saving the dbs for passwords, etc. The messages literally don't print and instead it just prints the "Custom command response was longer than 2k (contact an admin on the server...)" and nothing else. Only time I've seen it print anything else is when it's a not enough args passed which is handled by parseArgs.

Here are my config values:
{{$joinmessage := "You have been given your role back."}}
{{$leavemessage := "You are now on break. Please let us know if you need anything!!"}}
{{$staff := "Contact Esen, Samara, or Yog if there's an issue here."}}
{{$logchannel := 606354005156823041}}
{{$utcoffset := 0}}
{{$color := 0}}
{{$checking := 1}}
{{$access := `Administrator`}}
{{$roleid := 797340227927932948}}

Here's a picture of someone without the $roleid role trying to run the command. It posts the message but still posts the other 2k chars one as well.
![image](https://user-images.githubusercontent.com/52873478/106826505-e5d55880-663b-11eb-8abe-799c947e5e08.png)

**Comment by WickedWizard3588** [comment]
Ok, then, when I'm free can I get an invite to both the servers?

I'll DM you on Discord `WickedWizard#3588`

**Comment by WickedWizard3588** [comment]
Thanks for the invite :)

The issue is probably as you said, I'll push a fix when time permits. If you already have a fix, please PR it :)

**Comment by EsenDR** [comment]
Okay so did some more testing - the older version doesn't work either (same error as the new one). 

Decided to break it up into bits so start with only putting `-addstaff` command, and immediately errored to the same error. This is leading me to believe this is a (possibly) a server thing than a CC thing. This command, again, works fine in the test server.

**Comment by WickedWizard3588** [comment]
Ok, so I guess I need to agree with you. I made a few dummy servers, and it worked perfectly fine with me. It neither works with PAG, nor YAG. This means it shouldn't work with any other Selfhost of YAG. Can you try with any other self-host of YAG?


> Decided to break it up into bits so start with only putting `-addstaff` command, and immediately errored to the same error. This is leading me to believe this is a (possibly) a server thing than a CC thing. This command, again, works fine in the test server.

This makes me curious and maybe, we need to ask Jonas for this...........


> adding the command 1 subcommand at a time made the entire thing function as is.

Ok, let me tell you what I understood. I guess you mean putting each subcommand into different CC's. Like `addstaff` in one, `removestaff` in another, and so on. This brings me to a dumb conclusion that your guild is `special` and does not allow the usual limits. Still, check trying with any other YAG Self-host. 

Also, could I possibly get your Server ID? If any other issues pop up like this, it may be useful to compare the Server ID's and see what's common in between them.

**Comment by EsenDR** [comment]
> Can you try with any other self-host of YAG?
I'm not sure who else has a Self-Host of Yag but I can try and find out if the CC works for who does.

> I guess you mean putting each subcommand into different CC's.
Nope, I mean put it into the same CC just in pieces. What I did was put the first `addstaff` and added a `{{end}}` at the end, ran the command saw that it was working fine. Then removed the `{{end}}` and added `removestaff` and saw that it was working, and repeated the process until the entire command was there and functioning. Works perfectly as intended now, which I find super strange. I was seeing where white-spaces were becoming an issue.

> could I possibly get your Server ID?
Yeah the server ID is `434912082253709313`.

> This brings me to a dumb conclusion that your guild is `special` and does not allow the usual limits.
There have been problems like this in the past that I had to figure out ways to bypass lol. Like Pedro's birthday CC - he had to join the server and see what was up with it, and he concluded that it was an issue with the UTC offset until he realized it was an issue with how the CC was working in my server.
